content/en/docs/instrumentation/java/automatic/spring-boot.md

```diff
@@ -144,9 +144,432 @@ dependencies {
 
 {{% /tab %}} {{< /tabpane>}}
 
-### Additional instrumentations
 
-#### JDBC Instrumentation
+## Automatic instrumentation with Spring auto configurations
```

Comment:
I would make this section ### level and move it to the _OpenTelemetry Spring starter_ part